uqcs/templates/form.mako

```diff
@@ -1,176 +1,243 @@
+        <label for="memberType"
+          >Member Type <span class="reqstar">*</span></label
+        >
+        <div class="checkbox">
+          <label>
+            <input name="student" type="checkbox" id="studentCheckbox" /> Are
+            you currently a student?
+          </label>
+        </div>
       </div>
-      <div class="radio">
-        <label>
-          <input type="radio" value="postgrad" name="degreeType">
-          Postgrad
-        </label>
+      <div id="student-form-section" style="display:none;">
+        <div class="form-group">
+          <label for="student-no"
+            >Student Number <span class="reqstar">*</span></label
+          >
+          <input
+            type="number"
+            name="student-no"
+            class="form-control"
+            id="studentNo"
+            placeholder="43108765"
+          />
+          <p class="help-block">8 digits, no 's'</p>
+        </div>
+        <div class="form-group">
+          <label>Domestic or International</label> <br />
+          <div class="btn-group btn-group-toggle" data-toggle="buttons">
+            <label class="btn btn-primary">
+              <input type="radio" value="domestic" name="domORint" /> Domestic
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" value="international" name="domORint" />
+              International
+            </label>
+          </div>
+        </div>
+        <div class="form-group">
+          <label>Degree/Program</label>
+          <input
+            type="text"
+            name="degree"
+            class="form-control"
+            placeholder="e.g. BEng (Software)"
+          />
+        </div>
+        <div class="form-group">
+          <label>Degree Type</label> <br />
+          <div class="btn-group btn-group-toggle" data-toggle="buttons">
+            <label class="btn btn-primary">
+              <input type="radio" value="undergrad" name="degreeType" />
+              Undergraduate
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" value="postgrad" name="degreeType" /> Postgraduate
+            </label>
+          </div>
+        </div>
+        <div class="form-group">
+          <label>Year</label> <br />
+          <div class="btn-group btn-group-toggle" data-toggle="buttons">
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="1" /> 1
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="2" /> 2
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="3" /> 3
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="4" /> 4
+            </label>
+            <label class="btn btn-primary">
+              <input type="radio" name="year" value="5" /> 5
```

Comment:
should this be 5+, don't want to make @jsutton101 feel left out.

Reply:
Hahaha, good spot.